File: CS/Demo/WindowsApplication1/MyLayoutControl/MySkinViewInfoRegistrator.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraPrinting.Export;
using DevExpress.XtraTab.Registrator;
using DevExpress.XtraTab.ViewInfo;

namespace WindowsApplication1
{
    public class MySkinViewInfoRegistrator : SkinViewInfoRegistrator
    {

        public MySkinViewInfoRegistrator()
        {

        }

        public override DevExpress.XtraTab.ViewInfo.BaseTabHeaderViewInfo CreateHeaderViewInfo(DevExpress.XtraTab.ViewInfo.BaseTabControlViewInfo viewInfo)
        {
            return new MySkinTabHeaderViewInfo(viewInfo);
        }

        
        
    }
}

File: VB/WindowsApplication1/MyLayoutControl/MySkinViewInfoRegistrator.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraPrinting.Export
Imports DevExpress.XtraTab.Registrator
Imports DevExpress.XtraTab.ViewInfo

Namespace WindowsApplication1
	Public Class MySkinViewInfoRegistrator
		Inherits SkinViewInfoRegistrator

		Public Sub New()

		End Sub

		Public Overrides Function CreateHeaderViewInfo(ByVal viewInfo As DevExpress.XtraTab.ViewInfo.BaseTabControlViewInfo) As DevExpress.XtraTab.ViewInfo.BaseTabHeaderViewInfo
			Return New MySkinTabHeaderViewInfo(viewInfo)
		End Function



	End Class
End Namespace